core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -694,13 +694,13 @@ public static String getGenomicBuild(){
 
     public static String getLinkToPatientView(String caseId, String cancerStudyId)
     {
-        return "case.do#/patient?caseId=" + caseId
+        return "/patient?caseId=" + caseId
```

Comment:
i believe these need to relative b/c otherwise on instance nested in a subfolder, user will get taken to root

Reply:
@alisman seems like this is already being pasted after full url of instance (see references to `getLinkToPatientView`